CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ Which is based on [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
 
+## [1.4.1] - 2018-02-17
```

Comment:
Can you please remove the versioning info we will do this post acceptance, we do this for several reasons:
- There is no SLA on reviews
- reviews and merges are not FIFO
- maintainer may feel that it needs to be versioned differently

To avoid needing to have people rebase as often it is just better if we do that post acceptance.